### PR TITLE
fix GFM code blocks formatting in READMEs

### PR DESCRIPTION
**Pull request by rutsky**
According to GitHub Flavored Markdown [documentation](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks).

### CODE REVIEW

**Comment by rutsky** [comment]
Signed Google CLA.

**Comment by pphaneuf** [comment]
Didn't look at the change yet, but since this is your first contribution, you should add yourself to the `AUTHORS` and `CONTRIBUTORS` files as well.

**Comment by pphaneuf** [comment]
LGTM, I'll merge once you add yourself to the `AUTHORS`/`CONTRIBUTORS` files.

**Comment by rutsky** [comment]
@pphaneuf , I added myself to the list of contributors.

File: CONTRIBUTORS [hunk 1]
```diff
@@ -43,4 +43,5 @@ Paul Lietar <lietar@google.com>
 Pierre Phaneuf <pphaneuf@google.com>
 Rob Stradling <rob@comodo.com>
 Ruslan Kovalov <ruslan.kovalyov@gmail.com>
+Vladimir Rutsky <vladimir@rutsky.org>
```
Comment (pphaneuf) [inline CONTRIBUTORS]: Is your contribution owned by one of the companies already in `AUTHORS`? If not, add yourself to `AUTHORS`, as well (see the explanation at the top of this file).

Comment (rutsky) [inline CONTRIBUTORS]: Sorry, missed that part of `AUTHORS`. Done!

**Comment by pphaneuf** [comment]
Thanks!

**Pull request merged**